.circleci/lint_pr.sh

```diff
@@ -8,5 +8,5 @@ if [ -z "$PR_NUM" ]; then
 fi
 
 # get PR file list, filter out removed files, filter only JS/TS files, then pass to the linter
-curl -fsSL https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PR_REPONAME/pulls/$PR_NUM/files | jq -r '.[] | select(.status!="removed") | .filename' | grep -E '\.(js|jsx|ts|tsx)$' | xargs yarn eslint
+curl -fsSL https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PR_REPONAME/pulls/$PR_NUM/files | jq -r '.[] | select(.status!="removed") | .filename' | grep -E '\.(js|jsx|ts|tsx)$' | xargs --no-run-if-empty yarn eslint
```

Comment:
This PR triggered failure here.
The `--no-run-if-empty` skips if stdin is empty.

Tested also with https://github.com/aws-amplify/amplify-cli/pull/11946 to see if it still works with non empty file list 

![image](https://user-images.githubusercontent.com/5849952/217670977-501a684b-6441-4cd5-9b03-f97f09e29e80.png)